core/iwasm/common/wasm_exec_env.c

```diff
@@ -211,6 +211,7 @@ wasm_exec_env_set_thread_info(WASMExecEnv *exec_env)
     exec_env->handle = os_self_thread();
     exec_env->native_stack_boundary =
         stack_boundary ? stack_boundary + WASM_STACK_GUARD_SIZE : NULL;
+    exec_env->native_stack_top_min = (void *)UINTPTR_MAX;
```

Comment:
Had better add `#if WASM_ENABLE_MEMORY_PROFILING != 0`?

Reply:
as the wamrc part is separately controlled with a cli option, it's safer to initialize unconditionally here.